docusaurus/pnpm-lock.yaml

```diff
@@ -1,400 +1,5880 @@
-    version: 3.0.0(@types/react@18.2.46)(react@18.2.0)
-  async:
-    specifier: 2.6.4
-    version: 2.6.4
-  autoprefixer:
-    specifier: 10.4.16
-    version: 10.4.16(postcss@8.4.32)
-  classnames:
-    specifier: ^2.3.2
-    version: 2.3.2
-  clsx:
-    specifier: ^1.1.1
-    version: 1.1.1
-  copy-webpack-plugin:
-    specifier: 11.0.0
-    version: 11.0.0(webpack@5.89.0)
-  core-js:
-    specifier: 3.35.0
-    version: 3.35.0
-  css-declaration-sorter:
-    specifier: 6.3.0
-    version: 6.3.0(postcss@8.4.32)
-  css-minimizer-webpack-plugin:
-    specifier: 4.0.0
-    version: 4.0.0(webpack@5.89.0)
-  cssnano:
-    specifier: 6.0.2
-    version: 6.0.2(postcss@8.4.32)
-  cssnano-preset-advanced:
-    specifier: 6.0.2
-    version: 6.0.2(postcss@8.4.32)
-  del:
-    specifier: 6.1.1
-    version: 6.1.1
-  docusaurus-plugin-hubspot:
-    specifier: ^1.0.0
-    version: 1.0.0
-  docusaurus-plugin-segment:
-    specifier: ^1.0.3
-    version: 1.0.3
-  html-loader:
-    specifier: ^4.2.0
-    version: 4.2.0(webpack@5.89.0)
-  js-yaml:
-    specifier: ^4.1.0
-    version: 4.1.0
-  json-schema-faker:
-    specifier: ^0.5.4
-    version: 0.5.4
-  node-fetch:
-    specifier: ^3.3.2
-    version: 3.3.2
-  nth-check:
-    specifier: 2.0.1
-    version: 2.0.1
-  postcss-convert-values:
-    specifier: 6.0.1
-    version: 6.0.1(postcss@8.4.32)
-  postcss-discard-comments:
-    specifier: 6.0.1
-    version: 6.0.1(postcss@8.4.32)
-  postcss-loader:
-    specifier: 7.3.4
-    version: 7.3.4(postcss@8.4.32)(typescript@5.3.3)(webpack@5.89.0)
-  postcss-merge-longhand:
-    specifier: 6.0.1
-    version: 6.0.1(postcss@8.4.32)
-  postcss-merge-rules:
-    specifier: 6.0.2
-    version: 6.0.2(postcss@8.4.32)
-  postcss-minify-selectors:
-    specifier: 6.0.1
-    version: 6.0.1(postcss@8.4.32)
-  postcss-normalize-positions:
-    specifier: 6.0.1
-    version: 6.0.1(postcss@8.4.32)
-  postcss-normalize-repeat-style:
-    specifier: 6.0.1
-    version: 6.0.1(postcss@8.4.32)
-  postcss-ordered-values:
-    specifier: 6.0.1
-    version: 6.0.1(postcss@8.4.32)
-  prism-react-renderer:
-    specifier: ^2.3.1
-    version: 2.3.1(react@18.2.0)
-  react:
-    specifier: ^18.2.0
-    version: 18.2.0
-  react-dom:
-    specifier: ^18.2.0
-    version: 18.2.0(react@18.2.0)
-  react-markdown:
-    specifier: ^8.0.7
-    version: 8.0.7(@types/react@18.2.46)(react@18.2.0)
-  react-router:
-    specifier: 5.3.3
-    version: 5.3.3(react@18.2.0)
-  sanitize-html:
-    specifier: ^2.12.1
-    version: 2.12.1
-  sockjs:
-    specifier: 0.3.24
-    version: 0.3.24
-  trim:
-    specifier: 0.0.3
-    version: 0.0.3
-  unist-builder:
-    specifier: ^4.0.0
-    version: 4.0.0
-  unist-util-select:
-    specifier: ^5.1.0
-    version: 5.1.0
-  unist-util-visit:
-    specifier: ^5.0.0
-    version: 5.0.0
-  webpack-dev-server:
-    specifier: 4.9.2
-    version: 4.9.2(webpack@5.89.0)
-  yaml-loader:
-    specifier: ^0.8.0
-    version: 0.8.0
+importers:
+
+  .:
```

Comment:
@bnchrch do you know why there are so many changes to the lock file in this PR? The other changes look good to me but I wanted to understand why this is changing so much before approving

Reply:
I have no idea. I ran pnpm install 🤷 

I can revert that file no problem

Reply:
Hmm yeah, when I revert that file, and then run `pnpm install`, it produces no changes to the lock file.

Not sure why it is different for you but lets go ahead and revert it